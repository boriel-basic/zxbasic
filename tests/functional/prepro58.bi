sub test()
    asm
        ex af,af'               ; A'=player
    end asm
end sub

