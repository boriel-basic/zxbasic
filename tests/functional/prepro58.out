#line 1 "prepro58.bi"
sub test()
    asm
        ex af,af'               
    end asm
end sub

